Aug 29 15:10:02 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: === UAFT tg_runner started ===
Aug 29 15:10:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: Traceback (most recent call last):
Aug 29 15:10:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 60, in main
Aug 29 15:10:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:     res = AI.route(event)  # ожидаем dict
Aug 29 15:10:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: AttributeError: module 'ai_supervisor' has no attribute 'route'
Aug 29 15:10:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: Traceback (most recent call last):
Aug 29 15:10:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 60, in main
Aug 29 15:10:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:     res = AI.route(event)  # ожидаем dict
Aug 29 15:10:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: AttributeError: module 'ai_supervisor' has no attribute 'route'
Aug 29 15:10:26 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: Traceback (most recent call last):
Aug 29 15:10:26 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 60, in main
Aug 29 15:10:26 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]:     res = AI.route(event)  # ожидаем dict
Aug 29 15:10:26 ubuntu-s-1vcpu-1gb-sgp1-01 python[183979]: AttributeError: module 'ai_supervisor' has no attribute 'route'
Aug 29 15:12:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]: === UAFT tg_runner started ===
Aug 29 15:13:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]: Traceback (most recent call last):
Aug 29 15:13:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 60, in main
Aug 29 15:13:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]:     res = AI.route(event)  # ожидаем dict
Aug 29 15:13:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]: AttributeError: module 'ai_supervisor' has no attribute 'route'
Aug 29 15:13:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]: Traceback (most recent call last):
Aug 29 15:13:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 60, in main
Aug 29 15:13:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]:     res = AI.route(event)  # ожидаем dict
Aug 29 15:13:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[184091]: AttributeError: module 'ai_supervisor' has no attribute 'route'
